contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,80 @@
+[[contributing-user-stories]]
+= Contribute user stories to OpenShift documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for OpenShift GitHub repository
+
+toc::[]
+
+== Modularization back story
+OpenShift docs are getting modularized, starting from OpenShift 4.0.
+All existing content will be completely replaced with content that is based on user stories and
+complies with the modularization guidelines. All future content must support a user story.
+
+== How do I contribute user stories?
+Familiarize yourself with the Customer Content Services (CCS)
+link:https://redhat-documentation.github.io/modular-docs/[modularization guide]. This is an important
+first step in making sure you are helping the docs team create content that is fully modularized.
+
+== What is the branch setup for modularized docs?
+
+* link:https://github.com/openshift/openshift-docs/tree/enterprise-4.0[enterprise-4.0]
++
+This builds both OCP 4.0 and OKD/Upstream content. Just like today, OKD content will be separated using
+ifdef statements in content and distro statements in the _topic_map.yml.
++
+Only CCS submits and merges their PRs here untill OCP 4.0 is released, unless others are keen to submit
+content in modularized format.
+
+* https://github.com/openshift/openshift-docs/tree/master[master]
+This builds legacy (existing), mostly non user story based content. This branch will continue to update
+docs.okd.io (upstream) untill we switch to 4.0 with the enterprise-4.0 branch.
```

Comment:
quick q - am i right to believe/assume that following the switch to `enterprise-4.0` branch, the same branch will be used for OKD too ?

if so wouldn't make sense to not name it enterprise but something else to be common for OCP and OKD ?